test/spec/directives/env-var-test.spec.js

```diff
@@ -0,0 +1,391 @@
+'use strict';
+
+// these are set as globals in the .jshintrc file, but are first defined here
+// to eliminate repitition in the actual spec files
```

Comment:
I dont know if you need all these `global` annotations since these functions are in this file.